subtests/docker_cli/events/test_event_parsing.py

```diff
@@ -0,0 +1,167 @@
+# -*- python -*-
+#
+from unittest2 import TestCase, main        # pylint: disable=unused-import
+import autotest  # pylint: disable=unused-import
+from dockertest.output import DockerTime
+import events
+
+
+class TestEventParsing(TestCase):
+    def test_1_09(self):
+        """
+        docker-1.9 format: container events
+        """
+        t = ["2016-04-06T09:53:33.265109190-04:00",
+             "2016-04-06T09:53:38.048694595-04:00",
+             "2016-04-06T09:53:41.016729639-04:00"]
+        cid = ("39b75e2aef85774dc545acc9998f0c44"
+               "d0af5f1baf32617e8bc5ed5f998cc558")
+        source = "registry.access.redhat.com/rhel7/rhel:latest"
+
+        event_log = ("{t[0]} {cid}: (from {source}) create\n"
+                     "{t[1]} {cid}: (from {source}) start\n"
+                     "{t[2]} {cid}: (from {source}) die\n".format(**locals()))
+        keys = ['datetime', 'identifier', 'operation', 'source']
+        expect = [
+            (cid, dict(zip(keys, [DockerTime(t[0]), cid, 'create', source]))),
+            (cid, dict(zip(keys, [DockerTime(t[1]), cid, 'start',  source]))),
+            (cid, dict(zip(keys, [DockerTime(t[2]), cid, 'die',    source])))]
```

Comment:
OIC.  Yeah, better to do local disables then, just where it needs to be ignored.  I like to avoid globally disabling checks unless it really is absolutely necessary.